scanner/src/main/kotlin/experimental/Utils.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.scanner.experimental
+
+import org.ossreviewtoolkit.model.Provenance
+import org.ossreviewtoolkit.model.RepositoryProvenance
+
+/**
+ * Throw a [ScanStorageException] if [provenance] is a [RepositoryProvenance] with a non-empty VCS path.
+ */
+internal fun requireVcsPathBlank(provenance: Provenance) {
```

Comment:
Nit: Maybe "requireBlankVcsPath" would be a slightly more readable name.

Actually, there's also a minor inconsistency between docs and code in terms of blank vs. empty: I believe we should name this "requireEmptyVcsPath" and also make the check say `.isNotEmpty()`, because a VCS path consisting of spaces would also be a present VCS path.